From cbcb1973ab5117ce6c9bb48cca9edcb1fc8a8078 Mon Sep 17 00:00:00 2001
From: Alex Leibovich <alexl@marvell.com>
Date: Sun, 30 Jun 2019 15:45:43 +0300
Subject: [PATCH 317/386] dtsi: cn913x: mmc: removed no-1-8-v for CP0

This field was a temporary workaround, in order to avoid
"mmc1: 1.8V regulator output did not became stable" error
message.

Change-Id: Ic551d1e20ca8b79eff4487b7b36eb5d058379c58
Signed-off-by: Alex Leibovich <alexl@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/11830
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/cn9130-db.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/cn9130-db.dtsi b/arch/arm64/boot/dts/marvell/cn9130-db.dtsi
index ef5d58aa23e4..812ba7845f2f 100644
--- a/arch/arm64/boot/dts/marvell/cn9130-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/cn9130-db.dtsi
@@ -150,7 +150,6 @@
 		     &cp0_sdhci_cd_pins>;
 	bus-width = <4>;
 	cd-gpios = <&cp0_gpio2 11 GPIO_ACTIVE_LOW>;
-	no-1-8-v;
 	vqmmc-supply = <&cp0_reg_sd_vccq>;
 	vmmc-supply = <&cp0_reg_sd_vcc>;
 };
-- 
2.17.1

